### ISSUE TITLE
metrics: 2 barrier_latency_avg lines in Grafana

### ISSUE THREAD

**Issue by cyliu0** [risingwavelabs/risingwave#18535]
### Describe the bug

https://grafana.test.risingwave-cloud.xyz/d/EpkBw5W4k/risingwave-dev-dashboard?from=1726224671000&orgId=1&to=1726267968000&var-datasource=cdtasocg64074c&var-namespace=usrlngvty-20240913-104055&var-instance=benchmark-risingwave&var-pod=All&var-component=All&var-table=All&editPanel=16

![image](https://github.com/user-attachments/assets/3a3c6336-6cd7-4b00-b7be-3dacb340a4fb)

### The version of RisingWave

v2.0.0-rt.5 https://github.com/risingwavelabs/risingwave/commit/870790cad2604f1d0172c56eb2ba6c4206e988cb

**Comment by cyliu0** [comment]
It happened in v2.0.0-rt.4 https://github.com/risingwavelabs/risingwave/commit/8f9151257cdf6c5e48138526b39082aee7b2ffa1 too.

**Comment by fuyufjh** [comment]
<img width="1427" alt="Screenshot 2024-10-21 at 11 56 17" src="https://github.com/user-attachments/assets/6cf5d5f5-5f29-4f17-9b0b-469979ec88cc">

@cyliu0 There are 2 clusters in GCP and AWS with identical namespace. Is it expected?

**Comment by cyliu0** [comment]
Oh, I found that the two pipelines with the same namespace name in GCP and AWS.

https://buildkite.com/risingwave-test/longevity-test/builds/1854#0191eaf7-fc62-4586-95a6-51ec869f6bcc
https://buildkite.com/risingwave-test/longevity-test/builds/1855#0191eaf7-fa6b-406f-8010-8f0c72824563

The current release test automation workflow will assgin a random unique name to `BENCH_NAMESPACE` for every pipeline. But the longevity test doesn't use it like other pipelines. And we triggered multiple longevity tests for different purposes at the same time during release test. And the longevity test gave them the same namespace name. I will enhance the longevity test for this.

Close this one. Track it on https://github.com/risingwavelabs/risingwave-test/issues/775
